src/status_im2/contexts/syncing/setup_syncing/view.cljs

```diff
@@ -130,6 +114,20 @@
                :container-style {:margin-top 12}
                :icon-left       :i/copy}
               (i18n/label :t/copy-qr)]])]]
+        (when-not (sync-utils/valid-connection-string? @code)
+          ;; Implemented standard in-app authentication
+          [rn/view {:style style/standard-auth}
+           [standard-auth/view
+            {:track-text              (i18n/label :t/slide-to-reveal-code)
+             :thumb-color             profile-color
```

Comment:
we refer to the "users" color as `customization-color` across the codebase - this should be the same here 👍